docs/memcached.md

```diff
@@ -48,7 +20,7 @@ These are the metrics available for this scraper.
 | memcached.operations | Operation counts. | 1 | Sum | <ul> <li>type</li> <li>operation</li> </ul> |
 | memcached.threads | Number of threads used by the memcached instance. | 1 | Gauge | <ul> </ul> |
```

Comment:
Can we remove the html tags from the markdown?
1) It is consistent with other docs
2) HTML tags are not supported in Cloud Monitoring FE yet (we need to update the version of our client lib)

| Name | Description | Unit | Type | Attributes |
| ---- | ----------- | ---- | ---- | ---------- |
| memcached.bytes | Current number of bytes used by this server to store items. | By | Gauge | |
| memcached.commands | Commands executed. | 1 | Sum | command |
| memcached.connections.current | The current number of open connections. | connections | Gauge | |
| memcached.connections.total | Total number of connections opened since the server started running. | connections | Sum | |
| memcached.cpu.usage | Accumulated user and system time. | 1 | Sum | state |
| memcached.current_items | Number of items currently stored in the cache. | 1 | Gauge | |
| memcached.evictions | Cache item evictions. | 1 | Sum | |
| memcached.network | Bytes transferred over the network. | by | Sum | direction |
| memcached.operations | Operation counts. | 1 | Sum | type, operation |
| memcached.threads | Number of threads used by the memcached instance. | 1 | Gauge | |

Reply:
Fixed.